cities/schemas.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: Optional[str] = None
+
+
+class CityCreate(CityBase):
+    pass
+
+
+class CityUpdate(BaseModel):
+    additional_info: str
```

Comment:
The `CityUpdate` model currently only includes the `additional_info` field. Typically, update models should allow for partial updates, meaning all fields should be optional. Consider making `additional_info` optional or including other fields that might be updated.